### PR TITLE
[ansible/en-en] Ansible - request for a review

### PR DESCRIPTION
**Pull request by sirkubax**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

It is my first PR to learnxiny - please do not merge untill we settle :) I'd like to make one more update - but I need your review about overall content.

### CODE REVIEW

**Comment by sirkubax** [comment]
Oh - I would rebase next week - and do major cleanup - sorry for the mess.

Just now please focus on the overall text structure, and suggest improvements so I could work on them :)

**Comment by patriziotomato** [comment]
great work so far. thx for that!

File: ansible.html.markdown [hunk 1]
```diff
@@ -0,0 +1,600 @@
+---
+category: tool
+tool: ansible
+contributors:
+    - ["Jakub Muszynski" , "http://github.com/sirkubax"]
+filename: LearnAnsible.txt
+---
+
+## Ansible: the easiest orchestration tool
```
[the remark this answers is not in the document]
Comment (sirkubax) [reply ansible.html.markdown]: Thank you for the review - I'm working on it, though I'll need few more days. 
I will update asap :)

**Comment by divayprakash** [comment]
@sirkubax @so1gitworker Any progress on this?

**Comment by divayprakash** [comment]
@adambard Could you review this in its present state? If it's good enough to merge for now then I think it makes more sense to do that and then have the document for the community to take it forward.

**Comment by vendethiel** [comment]
It's in a good state for a beginning, I'd say.

**Pull request merged**